src/components/ReactPlayer/ReactPlayer.tsx

```diff
@@ -346,7 +353,18 @@ export const ReactPlayerBlock = React.forwardRef<ReactPlayerBlockHandler, ReactP
 
         return (
             <div
-                className={b({wrapper: !currentHeight, controls, started, hovered}, className)}
+                className={b(
+                    {
+                        wrapper: !currentHeight,
+                        controls,
+                        'background-shadow-rendered':
```

Comment:
what is the meaning of 'background-shadow-rendered' modifier?

Reply:
removed, as it is not needed anymore